dockerfiles/init/manifests/che.env

```diff
@@ -56,6 +56,13 @@
 #CHE_DEBUG_SUSPEND=false
 #CHE_DEBUG_PORT=8000
 
+# Che master logs
+#
+# Choose variant of Che master logs encoding and storing. Default is `plaintext`.
+# Another supported value is `json` which makes Che master produce JSON encoded logs
+# in system output of PID 1 process of container. Useful for storing logs in systems such as Logstash.
+# CHE_LOGS_APPENDERS_IMPL=plaintext
```

Comment:
if that is commented, default value will be used, do we have default value declared in a code? because I do not see it in the `che.props`

Reply:
I think the default value is declared [here](https://github.com/eclipse/che-lib/pull/83/files#diff-64fb0b389654f324dfc3abfe151c0b74R26)

Reply:
@l0rd is right this value is not read by the code and default value is in xml file of logback